test/Helper/HeadScriptTest.php

```diff
@@ -534,4 +534,19 @@ public function testSupportsAsyncAttribute()
         $test = $this->helper->__invoke()->toString();
         $this->assertContains('async="', $test);
     }
+
+    /**
+     * @group 23
+     */
+    public function testOmitsTypeAttributeIfNoneGivenAndHtml5Doctype()
+    {
+        $view = new \Laminas\View\Renderer\PhpRenderer();
+        $view->plugin('doctype')->setDoctype(\Laminas\View\Helper\Doctype::HTML5);
+        $this->helper->setView($view);
+
+        $this->helper->__invoke()->appendScript('// some script' . PHP_EOL);
+        $test = $this->helper->__invoke()->toString();
+        $this->assertNotContains('type="text/javascript"', $test);
+        $this->assertNotContains('type="TEXT/JAVASCRIPT"', $test);
```

Comment:
```suggestion
        $this->assertNotRegExp('#type="text/javascript#i"', $test);
```

A case-insensitive regexp is better here, as it ensures no case variation of that character sequence will be present.

Reply:
Thanks for your advises. I updated the test (+set your " to the correct position ;-) )